platform/broadcom/sai.mk

```diff
@@ -1,7 +1,7 @@
 LIBSAIBCM_XGS_VERSION = 10.1.7.0
-LIBSAIBCM_DNX_VERSION = 10.1.13.0
+LIBSAIBCM_DNX_VERSION = 10.1.21.0
 LIBSAIBCM_XGS_BRANCH_NAME = SAI_10.1.0_GA
-LIBSAIBCM_DNX_BRANCH_NAME = SAI_10.1.0_GA
+LIBSAIBCM_DNX_BRANCH_NAME = SAI_10.1.21.1
```

Comment:
why do we need to change the branch name?

Reply:
this is the azure storage path, this branch name is my local branch